src/coreComponents/constitutive/solid/ElasticIsotropic.cpp

```diff
@@ -58,6 +60,15 @@ ElasticIsotropic::ElasticIsotropic( string const & name, Group * const parent ):
   registerWrapper( viewKeyStruct::shearModulusString(), &m_shearModulus ).
     setApplyDefaultValue( -1 ).
     setDescription( "Elastic Shear Modulus Field" );
+
+  registerWrapper( viewKeyStruct::thermalExpansionCoefficientString(), &m_thermalExpansionCoefficient ).
+    setApplyDefaultValue( -1 ).
+    setDescription( "Thermal Expansion Coefficient Field" );
+
+  registerWrapper( viewKeyStruct::defaultThermalExpansionCoefficientString(), &m_defaultThermalExpansionCoefficient ).
+    setApplyDefaultValue( 0.0 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Default Thermal Expansion Coefficient" );
```

Comment:
There are thermal expansion coefficient of the solid rock frame and the thermal expansion coefficient of the fluid components. You may want to clarify this in the description?